addons/skin.confluence/720p/includes.xml

```diff
@@ -1,20 +1,19 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <includes>
-	<include file="defaults.xml"/>
-	<include file="ViewsVideoLibrary.xml"/>
-	<include file="ViewsMusicLibrary.xml"/>
-	<include file="ViewsFileMode.xml"/>
-	<include file="ViewsPictures.xml"/>
-	<include file="ViewsAddonBrowser.xml"/>
-	<include file="ViewsLiveTV.xml"/>
-	<include file="ViewsPVRGuide.xml"/>
-	<include file="ViewsWeather.xml"/>
-	<include file="IncludesCodecFlagging.xml"/>
-	<include file="IncludesHomeRecentlyAdded.xml"/>
-	<include file="IncludesHomeMenuItems.xml"/>
-	<include file="IncludesPVR.xml"/>
-	<include file="IncludesBackgroundBuilding.xml"/>
-
+	<include file="defaults.xml" />
+	<include file="ViewsVideoLibrary.xml" />
+	<include file="ViewsMusicLibrary.xml" />
+	<include file="ViewsFileMode.xml" />
+	<include file="ViewsPictures.xml" />
+	<include file="ViewsAddonBrowser.xml" />
+	<include file="ViewsLiveTV.xml" />
+	<include file="ViewsPVRGuide.xml" />
+	<include file="ViewsWeather.xml" />
+	<include file="IncludesCodecFlagging.xml" />
+	<include file="IncludesHomeRecentlyAdded.xml" />
+	<include file="IncludesHomeMenuItems.xml" />
+	<include file="IncludesPVR.xml" />
+	<include file="IncludesBackgroundBuilding.xml" />
 	<constant name="FanartCrossfadeTime">500</constant>
```

Comment:
hmm can I safely assume that constants are always used for animations? Or do i have to do them manually?